packages/js/src/core/fetch-token.ts

```diff
@@ -13,20 +13,17 @@ export interface FetchTokenByRefreshTokenParameters {
   scope?: string[];
```

Comment:
this should be non-nullable

Reply:
according to SDK Convention, this input parameter should be optional